dhall-csv/dhall-csv.cabal

```diff
@@ -0,0 +1,63 @@
+Name: dhall-csv
+Version: 0.0.1
+Cabal-Version: >=1.10
+Build-Type: Simple
+Tested-With: GHC == 8.2.2, GHC == 8.4.3, GHC == 8.6.1
```

Comment:
AFAIK we currently only have CI with GHC 8.6.5 and GHCJS. The other `Tested-With` stanzas are simply outdated. We should probably get rid of all of them, since we aren't doing a good job at keeping them up to date.

```suggestion
```

Reply:
Thanks! I wasn't really sure about that, I was using an old commit from dhall-docs to guide myself, I'll change it right away